source/removals.rst

```diff
@@ -0,0 +1,90 @@
+.. _removals:
+
+===========================
+ Other miscellaneous ideas
+===========================
+
+Dropping Python 2 support
+=========================
+
+With Python 2.7 reaching its supported end-of-life in 2020, like some other
+Python projects (e.g. IPython / Jupyter) we should seriously contemplate making
+pandas 2.0 only support Python 3.5 and higher. In addition to lowering the
```

Comment:
:+1: even if you decide not to do it for 2.0, I'd love to see pandas come up with a plan for going Python 3 only, and sign https://python3statement.github.io/